datafusion/physical-expr/src/aggregate/min_max.rs

```diff
@@ -1228,6 +1266,66 @@ mod tests {
         )
     }
 
+    #[test]
+    fn min_binary() -> Result<()> {
```

Comment:
I actually think the test in aggregate.slt will be sufficient -- we don't need these extra unit tests